experiment/src/client/SampleTypeDesigner/SampleTypeDesigner.tsx

```diff
@@ -14,73 +14,65 @@
  * limitations under the License.
  */
 
-import { List, Map } from "immutable";
-import * as React from 'react'
+import { Map } from "immutable";
+import React from 'react'
 import { ActionURL } from "@labkey/api";
 import {
     Alert,
+    BeforeUnload,
+    DomainDesign,
     DomainDetails,
     getSampleSet,
     getSampleTypeDetails,
-    IBannerMessage,
+    LoadingSpinner,
     SampleTypeDesigner,
-    SchemaQuery
+    SampleTypeModel
 } from "@labkey/components"
 
 import "@labkey/components/dist/components.css"
-import "./sampleTypeDesigner.scss";
-
-interface IAppState {
-    dirty: boolean
-    sampleType: DomainDetails
-    rowId: number
-    domainId?: number
-    messages?: List<IBannerMessage>,
-    queryName: string
-    returnUrl: string
-    schemaName: string
-    showConfirm: boolean
-    submitting: boolean
-    includeWarnings: boolean
-    showWarnings: boolean
-    badDomain: DomainDetails
+
+interface State {
+    sampleType?: DomainDetails
+    isLoading: boolean,
+    message?: string
     name?: string
     nameReadOnly?: boolean
 }
 
-const CREATE_SAMPLE_SET_ACTION = 'createSampleSet';
+const UPDATE_SAMPLE_SET_ACTION = 'updateMaterialSource';
+
+export class App extends React.PureComponent<any, State> {
 
-export class App extends React.PureComponent<any, Partial<IAppState>> {
+    private _dirty = false;
 
     constructor(props) {
         super(props);
 
-        const { RowId, schemaName, queryName, domainId, returnUrl, name, nameReadOnly } = ActionURL.getParameters();
+        const { name, nameReadOnly } = ActionURL.getParameters();
         const action = ActionURL.getAction();
 
-        let messages = (action !== CREATE_SAMPLE_SET_ACTION) ?
-            this.checkUpdateActionParameters( RowId,) :
-            List<IBannerMessage>();
+        let message;
+        if (action === UPDATE_SAMPLE_SET_ACTION && !this.getRowIdParam()) {
+            message = 'RowId parameter not supplied, unable to determine which Sample Type to edit.';
```

Comment:
```suggestion
            message = 'RowId parameter not supplied. Unable to determine which Sample Type to edit.';
```
Are we using Sample Type in LKS now instead of Sample Set?

Reply:
Nope, good catch. I'll change the usages / text in this component back to "Sample Set".